### ISSUE TITLE
Generalize wsgi FileWrapper so it also works in asgi

### ISSUE THREAD

**Issue by Kurocon** [Inter-Actief/amelie#672]
We use a WSGI FileWrapper to send files directly from a Django view in two places:

For the HttpResponseSendfile response (used in profile pictures)
https://github.com/Inter-Actief/amelie-private/blob/5c222eb76efa665ac61d4e4c33844e4d65755fe9/amelie/tools/http.py#L41

In the members.views.picture view (which is unused I believe, so this view could just be removed)
https://github.com/Inter-Actief/amelie-private/blob/6753e48c6c3a47c57f50a644251b6df291de8f5c/amelie/members/views.py#L1292

This won't work any more under the newer ASGI standard which will be used in the future, so we need a more general way to do this.

As an example, something like what is used to retrieve the thumbnails in the new photo uploader would work fine I think:
https://github.com/Inter-Actief/amelie-private/blob/6753e48c6c3a47c57f50a644251b6df291de8f5c/amelie/activities/views.py#L735

**Comment by Kurocon** [comment]
I just ran into this again when preparing to deploy Amelie to Kubernetes, for which I've tried to use daphne / asgi. This needs to be fixed before we can move over to Kubernetes.

PS. Funnily enough this issue is the 5th hit on DuckDuckGo when searching "wsgiref.util.FileWrapper asgi", which is how I re-found this :laughing:

**Comment by supertom01** [comment]
No longer relevant since we will not move to asgi.
